src/assets/theming/_base.scss

```diff
@@ -4,154 +4,108 @@
 }
 
 html {
-  background-color: $bg-clr;
-  color: $base-clr;
-  font-family: $main-font;
   font-size: 50%;
   scroll-behavior: smooth;
 
   /* Better Font Rendering =========== */
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
 
-
   @include breakpoint-md {
     font-size: 62.5%;
   }
 }
 
 body {
   box-sizing: border-box;
-  display: flex;
-  flex-direction: column;
-  justify-content: space-between;
   font-size: 2.25rem;
   height: 100vh;
   line-height: 140%;
   min-width: 320px;
-  margin: 0 5em 0 1.5em;
+  a {
+    text-decoration: none;
+    color: inherit;
+  }
 
   @include breakpoint-md {
     font-size: 1.8rem;
   }
-
-  footer {
-    justify-self: flex-end;
-  }
-}
-
-h1, h2, h3, h4, h5, h6 {
-  font-family: $accent-font;
-  line-height: 140%;
-  margin: 0 0 0.7em 0;
-}
-
-h1 {
-  font-size: 4rem;
-}
-
-a {
-  color: $link-clr;
-  transition: color 400ms;
-
-  &:hover {
-    color: $link-hvr-clr;
-  }
 }
 
-p {
-  max-width: 600px;
-  + p {
-    margin-top: 1.5rem;
-  } 
-}
-
-nav {
-  ul {
-    list-style: none;
+header {
+  background-color: $orange;
+  display: flex;
+  justify-content: space-between;
+  height: 4em;
+
+  div {
```

Comment:
it might be better to use the .logo class instead of div here in case you ever expand the header and need to add another div